app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
```

Comment:
- [ ] The view should be modified to match the wireframes and their styling.

### Your design

![Screenshot from 2022-02-02 15-05-58](https://user-images.githubusercontent.com/22434660/152129216-6bc87a50-ebf2-43c5-bfea-8cbcb79f1047.png)


### Given design
![Screenshot from 2022-02-01 18-46-05](https://user-images.githubusercontent.com/22434660/152129170-ed89e647-6f76-4f85-a8e3-b6d2ecc47cb4.png)

Reply:
Thank you for the feedback @Dipeshtwis . This will be implemented right away.

Reply:
![login_blog](https://user-images.githubusercontent.com/67757001/152146445-313dda0d-577f-4ab2-941c-7e600575cc02.png)
![sigup_blog](https://user-images.githubusercontent.com/67757001/152146461-1a316acc-5ad6-4141-8e6e-a033f328ddda.png)

Reply:
Done! 👍